src/zenml/zen_stores/migrations/versions/479103df60b6_add_triggers.py

```diff
@@ -0,0 +1,161 @@
+"""add triggers [479103df60b6].
+
```

Comment:
You've really thought of everything -  I was getting ready to do this one once your branch was merged.